ROMFS/px4fmu_common/init.d/airframes/17002_TF-AutoG2

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+#
+# @name ThunderFly Auto-G2
+#
+# @type Autogyro
+# @class Autogyro
+#
+# @output MAIN1 rotor_head_L
+# @output MAIN2 rotor_head_R
+# @output MAIN3 elevator
+# @output MAIN4 rudder
+# @output MAIN5 rudder (second, optional)
+# @output MAIN6 throttle
+# @output MAIN7 wheel
+#
+# @output AUX1 feed-through of RC AUX1 channel for prerotator (optional)
+# @output AUX2 feed-through of RC AUX2 channel for release device (optional)
+#
+# @url https://github.com/ThunderFly-aerospace/Auto-G2/
+# @maintainer ThunderFly s.r.o., Roman Dvorak <dvorakroman@thunderfly.cz>
+#
+
+sh /etc/init.d/rc.fw_defaults
+
+if [ $AUTOCNF = yes ]
+then
+	param set BAT_CAPACITY 2500.0
+	param set BAT_N_CELLS 3
+
+	param set PWM_AUX_RATE 50
+	param set PWM_RATE 50
+
+    param set SENS_BOARD_ROT 8
```

Comment:
Whitespace: can you use tabs?

Reply:
repaired.
Thanks for the warning. I didn't know about this convention.